presto-jdbc/src/main/java/com/facebook/presto/jdbc/AbstractInfo.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.jdbc;
+
+import com.facebook.presto.spi.type.TypeSignature;
+
+import java.sql.Types;
+
+abstract class AbstractInfo
+{
+    protected static final int VARCHAR_MAX = 1024 * 1024 * 1024;
```

Comment:
Let's leave all of these in `ColumnInfo` and simply change the visibility. It would be fine to move them to a shared utility class, but that refactoring should be done as a separate commit, so as not to distract from the primary purpose of this change (adding prepared statement metadata).

Alternatively, do the refactoring as a separate commit before the prepared statement commit (you can put both commits in the same Pull Request).